### PR TITLE
Fixes #3 and allows users to correctly reference their googleplus profiles in the template.

### LINKED ISSUE #3: Google Plus Social Icon Links to plus.google.com Regardless of Specified Username
Within the icons.html template, the googleplus variable is incorrectly referenced as google_plus.
I think it's a simple change to correct this to googleplus.

### PR DESCRIPTION
**Pull request by keyphact**

### CODE REVIEW

File: layouts/partials/icons.html [hunk 1]
```diff
@@ -6,6 +6,14 @@
 </li>
 {{ end }}
 
+{{ with .Site.Params.social.homepage }}
+<li>
+	<a href="mailto:{{ $.Site.Params.social.homepage }}" title="{{ $.Site.Data.l10n.homepage }}">
```
Comment (digitalcraftsman) [inline layouts/partials/icons.html:11]: I don't think that `mailto:` was intended in the href attribute, right?
Furthermore, the link should link to the base url and not to the value of `.Site.Params.social.homepage`. I already fixed both things.

**Comment by digitalcraftsman** [comment]
Hello @keyphact,

thank you for contributing to this theme. I've merged you additions as a8eb701e7b6ef0e8804441e6e5bd858adaa8f8e6.

Cheers,
Digitalcraftsman

File: layouts/partials/icons.html (hunk 1 above)
Comment (keyphact) [inline layouts/partials/icons.html:11]: No was not intended, thanks for fixing!

**Comment by keyphact** [comment]
Pleasure.

**Pull request closed without merging**